### PR TITLE
docs(ui5-dialog): add documentation for keyboard handling

### LINKED ISSUE #8190: [SF_ACC][ui5-dialog]: CNTL + Arrow keys not working for draggable dialog
### Bug Description

CNTL + Arrow keys are not working for draggable dialog to move the dialog.
Where is it documented for how to resize and move dialogs using the keyboard? We need to tell the customer.

### Affected Component

_No response_

### Expected Behaviour

CNTL + Arrow key should be working for draggable dialog to move the dialog.
You can ui5 page https://ui5.sap.com/#/entity/sap.m.Dialog/sample/sap.m.sample.Dialog, it works.

### Isolated Example

_No response_

### Steps to Reproduce

1. open page https://sap.github.io/ui5-webcomponents/playground/?path=/story/main-dialog--draggable-and-resizable
2. Open the dialog
3. When the dialog is opened, the focus is on the dialog title bar by default. Use CNTL+arrow keys and can't move the dialog.

### Log Output, Stack Trace or Screenshots

_No response_

### Priority

Medium

### UI5 Web Components Version

1.21.x

### Browser

Chrome, Edge, Firefox, Safari

### Operating System

IOS

### Additional Context

_No response_

### Organization

SAP SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

### PR DESCRIPTION
**Pull request by s-todorova**
Missing information about keyboard shortcuts for dragging/resizing the dialog is now properly documented.

Fixes #8190

### CODE REVIEW

File: packages/main/src/Dialog.ts [hunk 1]
```diff
@@ -79,6 +79,25 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
  * <br>
  * For more information see the sample "Bar in Header/Footer".
 
+ *<h3>Keyboard Handling</h3>
+ *
+ * <h4>Basic Navigation</h4>
+ * When the <code>ui5-dialog</code> is focused and has the <code>draggable</code> property set, the user can move the dialog
```
Comment (LidiyaGeorgieva) [inline packages/main/src/Dialog.ts:85]: The focus  should be on the header in order to use the keys to drag the dialog.
I suggest:
When the header of the code>ui5-dialog</code> is focused.

But in this case we should say first that the dialog is draggable (if its not - the header can't be focused :) ).
So this:
"When the <code>ui5-dialog</code> is focused and has the <code>draggable</code> property set"
could be said as this:
"When the <code>ui5-dialog</code> has the <code>draggable</code> property set and the header is focused"

File: packages/main/src/Dialog.ts [hunk 2]
```diff
@@ -79,6 +79,25 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
...
+ * with the following keyboard shortcuts:
+ * <br>
+ * <ul>
+ * <li>[UP/DOWN] arrow keys - Move the dialog to the top/bottom.</li>
+ * <li>[LEFT/RIGHT] arrow keys - Move the dialog to the left/right.</li>
+ * </ul>
+ *
+ * <h4>Resizing</h4>
+ * When the <code>ui5-dialog</code> is focused and has the <code>resizable</code> property set, the user can change the size of the dialog
```
Comment (LidiyaGeorgieva) [inline packages/main/src/Dialog.ts:94]: same as above

File: packages/main/src/Dialog.ts [hunk 3]
```diff
@@ -79,6 +79,25 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
...
  * For more information see the sample "Bar in Header/Footer".
 
+ *<h3>Keyboard Handling</h3>
+ *
+ * <h4>Basic Navigation</h4>
+ * When the <code>ui5-dialog</code> is focused and has the <code>draggable</code> property set, the user can move the dialog
+ * with the following keyboard shortcuts:
+ * <br>
+ * <ul>
+ * <li>[UP/DOWN] arrow keys - Move the dialog to the top/bottom.</li>
```
Comment (LidiyaGeorgieva) [inline packages/main/src/Dialog.ts:89]: up/down

File: packages/main/src/Dialog.ts [hunk 4]
```diff
@@ -79,6 +79,25 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
...
+ * <li>[LEFT/RIGHT] arrow keys - Move the dialog to the left/right.</li>
```
Comment (LidiyaGeorgieva) [inline packages/main/src/Dialog.ts:90]: left/right

File: packages/main/src/Dialog.ts [hunk 5]
```diff
@@ -79,6 +79,25 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
  * <br>
  * For more information see the sample "Bar in Header/Footer".
 
+ *<h3>Keyboard Handling</h3>
+ *
+ * <h4>Basic Navigation</h4>
+ * When the <code>ui5-dialog</code> has the <code>draggable<code> property set and the header is focused, the user can move the dialog
```
Comment (BorisDafov) [inline packages/main/src/Dialog.ts:85]: Better:
When the <code>ui5-dialog</code> has the <code>draggable<code> property set to <code>true<code> and the header is focused, the user can move the dialog

File: packages/main/src/Dialog.ts [hunk 6]
```diff
@@ -79,6 +79,25 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
...
+ * with the following keyboard shortcuts:
+ * <br>
+ * <ul>
+ * <li>[UP/DOWN] arrow keys - Move the dialog up/down.</li>
+ * <li>[LEFT/RIGHT] arrow keys - Move the dialog left/right.</li>
+ * </ul>
+ *
+ * <h4>Resizing</h4>
+ * When the <code>ui5-dialog</code> has the <code>resizable<code> property set and the header is focused, the user can change the size of the dialog
```
Comment (BorisDafov) [inline packages/main/src/Dialog.ts:94]: same:
When the <code>ui5-dialog</code> has the <code>resizable<code> property set to <code>true<code> and the header is focused, the user can change the size of the dialog

**Review by BorisDafov — CHANGES_REQUESTED** [review]

**Comment by BorisDafov** [comment]
I approve the change from UA perspective.

**Review by LidiyaGeorgieva — APPROVED** [review]

**Pull request merged**